doc/nrf/releases_and_maturity/releases/release-notes-2.9.0-nrf54h20-rc1.rst

```diff
@@ -0,0 +1,432 @@
+.. _ncs_release_notes_290-nrf54h20-rc1:
+
+|NCS| v2.9.0-nRF54H20-rc1 Release Notes
+#######################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+The |NCS| v2.9.0-nrf54h20-rc1 is a nRF54H20-exclusive release tag, only supporting products based on the nRF54H20 SoC.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+Added the following features as supported:
+
+* nRF54H20 Series:
+
+  * Triggering firmware recovery by pressing a hardware button or calling the respective API.
+  * Dynamic Global Domain Frequency Scaling (DGFS), optimizing USB suspend power consumption.
+  * Drivers adapted for the following peripherals:
+
+    * UARTE120
+    * PWM120
+    * SPIM120
+    * EXMIF
+
+  * UICR validation support.
+  * MSPI EXMIF support.
+
+* Power Management:
+
+  * System-On-All-Idle power improvements.
+  * Thread/Matter power efficiency improvements.
+
+Added the following features as experimental:
+
+* Bluetooth:
+
+  * :ref:`Parallel scanning and initiating connection <bt_scanning_while_connecting>`.
+    It reduces the overall waiting time when there are several Bluetooth devices to be discovered and connected at the same time.
+
+Improved the following features:
+
+* Better BICR handling and generation.
+* Enhanced power management features across various subsystems.
+
+Implemented the following bug fixes:
+
+* NCSDK-30802: Resolved an issue where the nRF54H20 device suddenly stopped transmitting ESB packets after nrfxlib 3.9.0.
+* NCSDK-30161: Fixed an assertion during boot time caused by a combination of :kconfig:option:`CONFIG_ASSERT`, :kconfig:option:`CONFIG_SOC_NRF54H20_GPD`, and external flash.
+* NCSDK-30117: Ensured that declaring a MEM component pointing to a memory region not assigned to a particular core is no longer possible.
+* NCSDK-29682: Added support for the `cose-alg-sha-512` algorithm in the SUIT module.
+
+Limitations
+***********
+
+On the nRF54H20 SoC, The device firmware update (DFU) procedure from external flash memory does not work with the new flash memory driver for XIP.
```

Comment:
The driver is not "for XIP". It uses the MSPI EXMIF driver to communicate with the flash chip, so I think this should be something like "... the new flash memory driver based on the MSPI EXMIF handling".